From 0f1272d8f6b9d4b7a3c947cf286322fe4aa1abdf Mon Sep 17 00:00:00 2001
From: Andrew Price <anprice@redhat.com>
Date: Thu, 9 Oct 2014 11:18:45 -0400
Subject: [redhat] configs: enable gfs2 for s390x

Message-id: <1412853525-23244-1-git-send-email-anprice@redhat.com>
Patchwork-id: 97298
O-Subject: [PATCH RHEL7.1] redhat/configs: enable gfs2 for s390x
Bugzilla: 1112658
RH-Acked-by: Steven Whitehouse <swhiteho@redhat.com>
RH-Acked-by: Robert S Peterson <rpeterso@redhat.com>
RH-Acked-by: Abhijith Das <adas@redhat.com>

Enable CONFIG_GFS2_FS (=m) in s390x builds.

Resolves: rhbz#1112658

Signed-off-by: Andrew Price <anprice@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/s390x/CONFIG_GFS2_FS b/redhat/configs/generic/s390x/CONFIG_GFS2_FS
deleted file mode 100644
index 5f66014..0000000
--- a/redhat/configs/generic/s390x/CONFIG_GFS2_FS
+++ /dev/null
@@ -1 +0,0 @@
-# CONFIG_GFS2_FS is not set
-- 
1.7.1